src/components/Business.tsx

```diff
@@ -1,5 +1,40 @@
-import { Card } from "react-bootstrap";
-import { BusinessEntry } from "../types/BusinessTypes";
+import { Card } from 'react-bootstrap';
+import '../App.css';
+import valueData from '../data/values.json';
+import { BusinessEntry } from '../types/BusinessTypes';
+import { getActiveColor } from './AdvancedSearch';
+
+const buttonStyle: React.CSSProperties = {
+  border: 'none',
+  borderRadius: '16px',
+  backgroundColor: '#ddd',
+  color: 'black',
+  paddingLeft: '7px',
+  paddingRight: '7px',
+  paddingTop: '3px',
+  paddingBottom: '3px',
+  marginLeft: '5px',
+  textAlign: 'center',
+  fontSize: 'small',
+};
+
+const dictionary: { [key: string]: string[] } = valueData;
+
+const getCoreValue = (subvalue: string) => {
```

Comment:
maybe this function can be moved into `filtering.tsx`
it might be useful in other files in the future and then you also won't need to import from `values.json`